styles.css

```diff
@@ -25,12 +25,26 @@ main {
   justify-content: space-around;
 }
 
+.clothes-title{
+  margin-left: 0.75em;
+}
+
+.favoriteStyled{
+  display: inline-block;
+  cursor: pointer;
+  font-size: .75em;
+  height: 15%;
+  width: 15%;
+  margin: 0em 1em 1em 1em;
+  border-radius: 45%;
+}
```

Comment:
We may have to rename this class to be more specific.  Doing this will help pinpoint classes in the html easier.

Reply:
Agreed! Solid suggestion.